diff --git a/src/script/script.pro b/src/script/script.pro
index 80e9b8a..f6040d7 100644
--- a/src/script/script.pro
+++ b/src/script/script.pro
@@ -13,7 +13,10 @@ DEFINES   += QT_NO_USING_NAMESPACE
 DEFINES   += QLALR_NO_QSCRIPTGRAMMAR_DEBUG_INFO
 #win32-msvc*|win32-icc:QMAKE_LFLAGS += /BASE:0x66000000       ### FIXME
 
-unix|win32-g++*:QMAKE_PKGCONFIG_REQUIRES = QtCore
+unix|win32-g++* {
+    QMAKE_PKGCONFIG_REQUIRES = QtCore
+    CONFIG += create_prl create_pc
+}
 
 load(qt_module_config)
 
diff --git a/src/scripttools/scripttools.pro b/src/scripttools/scripttools.pro
index 7ffc736..bf6968c 100644
--- a/src/scripttools/scripttools.pro
+++ b/src/scripttools/scripttools.pro
@@ -11,7 +11,10 @@ DEFINES   += QT_BUILD_SCRIPTTOOLS_LIB
 DEFINES   += QT_NO_USING_NAMESPACE
 #win32-msvc*|win32-icc:QMAKE_LFLAGS += /BASE:0x66000000
 
-unix|win32-g++*:QMAKE_PKGCONFIG_REQUIRES = QtCore QtGui QtScript
+unix|win32-g++* {
+    QMAKE_PKGCONFIG_REQUIRES = QtCore QtGui QtScript
+    CONFIG += create_prl create_pc
+}
 
 load(qt_module_config)
 
